src/Laravel/PassportServiceProvider.php

```diff
@@ -49,7 +49,7 @@ public function makeAuthorizationServer(): AuthorizationServer
             new ClaimExtractor(...$claimSets),
             Configuration::forSymmetricSigner(
                 app(config('openid.signer')),
-                InMemory::file($cryptKey->getKeyPath()),
+                InMemory::plainText($cryptKey->getKeyContents(), $cryptKey->getPassPhrase() ?? '')
```

Comment:
Isn't something like this safer? 

```suggestion
               file_exists($cryptKey->getKeyPath()) ?  InMemory::file($cryptKey->getKeyPath()) : 
                InMemory::plainText($cryptKey->getKeyContents(), $cryptKey->getPassPhrase() ?? '')
```

Reply:
My thinking was that Passport is anyway going to load the key, might as well ask it for the contents and defer logic on where the key comes from, to 

Might giving me as hint why your solution is safer?

Reply:
My option just make the check if the file exists before going to the fallback and your solution. It's a way to avoid any regression in the code.

I'll make some checks on my side to validate this (It seems this project gained some users interest, so I need to keep it stable).

I'll keep you updated ;)